vm-linuxagent-upgrade/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Create a Linux Virtual Machine with new version of Azure Linux Agent using Custom Script Linux Extension",
```

Comment:
Suggestion for name: Linux VM with new version of Azure Linux Agent using custom script Linux extension so that the text fits more nicely in the Azure.com index title column